.github/workflows/wpt-test.yml

```
@@ -1,7 +1,7 @@
 name: Tests
 
 on:
-  pull_request_target:
+  pull_request:
```

Ok, so that's how we test it.
Just revert event name and ready to merge.

```suggestion
  pull_request_target:
```

P.S. had to remove test description so that we don't have a failing check.